docs/wxm-devices/D1-bundle/D1-connect-and-claim.mdx

```diff
@@ -0,0 +1,233 @@
+---
+id: D1-connect-and-claim
+title: Connect and Claim Station
+sidebar_label: Connect and Claim Station
+slug: /wxm-devices/D1-bundle/D1-connect-and-claim
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+## WG1200 Gateway
+
+<img object-fit="contain" src={useBaseUrl("/img/home/wg1200-gateway.png")} />
+
+The weather station comes together with the WG1200 Gateway.
+The WG1200 Gateway receives all the meteorological data plus the location of the weather station, which are measured and transmitted every 16 seconds by the outdoor sensor
+and displays them on its screen.
+
+
+In this chapter, you will find guidance to set up the WG1200 Gateway.
+<hr/>
+
+## Prerequisites
+
+1. The station must be fully assembled (parts and batteries).
+1. A working smartphone with connection to the internet.
+1. The WeatherXM app installed on your smartphone. [WeatherXM Application](/wxm-devices/D1-bundle/D1-connect-and-claim#download-weatherxm-app-and-login)
+1. A reliable 2.4 GHz WiFi network (**5 GHz is not supported**) for the WG1200 Gateway to connect to.
```

Comment:
The WiFi should be above the app, the app should be the last one. Also as in my other comments, having "WeatherXM Application" after the dot feels not human. Just hyperlink the "WeatherXM app" in your sentence.